boards/shields/wiznet_wiz550io/wiznet_wiz550io.overlay

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2024 Grant Ramsay
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
add the following here:
```
/ {
        zephyr,user {
                wiz550io-reset-gpios = <&arduino_header 14 GPIO_ACTIVE_HIGH>;
                wiz550io-ready-gpios = <&arduino_header 13 GPIO_ACTIVE_HIGH>;
        };
};
```
and get them from the shield code like
```
const struct gpio_dt_spec reset_pin = GPIO_DT_SPEC_GET(DT_PATH(zephyr_user), wiz550io_reset_gpios);
```

Reply:
I wasn't aware of `zephyr,user` node, I prefer it to GPIO hogs.

Although, looking at the documentation, I doubt it would be allowed:

> This node is meant for sample code and user applications. It should not be used in the upstream Zephyr source code for device drivers, subsystems, etc.

Reply:
Using a binding for the board seems appropriate :) Please avoid introducing the `wiz550io-` part of the gpios properties for the board, that was only required to avoid name collisions in the `zephyr,user` node.